firefox-ios/Client/Frontend/Reader/ReaderModeHandlers.swift

```diff
@@ -116,8 +116,14 @@ struct ReaderModeHandlers: ReaderModeHandlersProtocol {
               let response = GCDWebServerDataResponse(html: html) else { return nil }
         // Apply a Content Security Policy that disallows everything except images from
         // anywhere and fonts and css from our internal server
-        response.setValue("default-src 'none'; img-src *; style-src http://localhost:* '\(ReaderModeStyleHash)'; font-src http://localhost:*",
-                          forAdditionalHeader: "Content-Security-Policy")
+        let csp = """
```

Comment:
Hello! Would you mind making that same change into `WKReaderModeHandlers` as well? I think that's the only adjustment missing for he `WebEngine` (not everything about reader mode is migrated in there, I am working on it).
 
Also, I think we can entirely remove the constant `ReaderModeStyleHash` with this change? 

Also wanted to highlight that there's a [comment](https://github.com/mozilla-mobile/firefox-ios/blob/9c2d26956f7cc45f53b25c2671b1b32ccb380c08/firefox-ios/Client/Frontend/UserContent/UserScripts/MainFrame/AtDocumentEnd/FindInPage.js#L18-L33) under the FindInPage.js that I'm unsure about that relates to this `ReaderModeStyleHash` constant. You might understand better than me what this does and if we can clean that up 😄